.github/workflows/docker-publish.yml

```
@@ -44,6 +44,9 @@ jobs:
 
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
+        with:
+          provenance: false
```

actionlint (1.7.11)

[error] input "provenance" is not defined in action "docker/setup-buildx-action@v3". available inputs are "append", "buildkitd-config", "buildkitd-config-inline", "buildkitd-flags", "cache-binary", "cleanup", "config", "config-inline", "driver", "driver-opts", "endpoint", "install", "keep-state", "name", "platforms", "use", "version"

(action)